index.js

```diff
@@ -49,6 +49,16 @@ function create(win, opts) {
 					download(win, props.srcURL);
 				}
 			}, {
+				id: 'copyImageLink',
+				label: 'Copy Image Link',
```

Comment:
Should be named `Copy Image Address` to match browsers. Same with all other name references.